src/app.js

```diff
@@ -91,6 +91,9 @@ app.post('/nominations', async (req, res) => {
     })
     await saveKueJob(createJob.attempts(3))
 
+    let signupTemplate = 'bnc-nomination'
+    await mail.sendEmailTemplate(body.nominatorEmail, 'Thanks for nominating a candidate.', signupTemplate, { name: 'Friend' })
```

Comment:
We actually have the nominator's name here, so we should use that instead of 'Friend'.